float64 mean_error
float64 MSE
float64 sigma
float64 indices